File: server-aspnet-fastendpoints/src/Org.OpenAPITools/Models/AssetResponse.cs
namespace Org.OpenAPITools.Models;


/// <summary>
/// 
/// </summary>
public class AssetResponse 
{
    public bool Valid { get; set; }
    public string Message { get; set; }
    public double VarVersion { get; set; }
    public bool SerializeNulls { get; set; }
    public long StartTimeLog { get; set; }
    public string ErrorCode { get; set; }
    public List<NameStringValueResponse> Request { get; set; }
    public long AssetId { get; set; }
    public string FileName { get; set; }
    public string Caption { get; set; }
    public string MediaType { get; set; }
    public string ThumbnailURL { get; set; }
    public string CoverURL { get; set; }
    public string FullURL { get; set; }
    public string AssetVersion { get; set; }
    public string VersionName { get; set; }
    public int VersionCode { get; set; }
    public string Display { get; set; }
    public string LocationDescription { get; set; }
    public AccountShortResponse Uploader { get; set; }
    public long LikesCount { get; set; }
    public int CommentsCount { get; set; }
    public double Latitude { get; set; }
    public double Longitude { get; set; }
    public string AttachedMediaType { get; set; }
    public string AttachedAssetURL { get; set; }
    public long Created { get; set; }
    public long Updated { get; set; }
    public string SearchTags { get; set; }
    public string MetaData { get; set; }
    public string Returning { get; set; }
}

File: server-scala-play-server/app/model/AssetResponse.scala
package model

import play.api.libs.json._

/**
  * Represents the Swagger definition for AssetResponse.
  */
@javax.annotation.Generated(value = Array("org.openapitools.codegen.languages.ScalaPlayFrameworkServerCodegen"), date = "2026-02-09T20:57:39.376804970Z[Etc/UTC]", comments = "Generator version: 7.20.0-SNAPSHOT")
case class AssetResponse(
  valid: Option[Boolean],
  message: Option[String],
  version: Option[Double],
  serializeNulls: Option[Boolean],
  startTimeLog: Option[Long],
  errorCode: Option[String],
  request: Option[List[NameStringValueResponse]],
  assetId: Option[Long],
  fileName: Option[String],
  caption: Option[String],
  mediaType: Option[String],
  thumbnailURL: Option[String],
  coverURL: Option[String],
  fullURL: Option[String],
  assetVersion: Option[String],
  versionName: Option[String],
  versionCode: Option[Int],
  display: Option[String],
  locationDescription: Option[String],
  uploader: Option[AccountShortResponse],
  likesCount: Option[Long],
  commentsCount: Option[Int],
  latitude: Option[Double],
  longitude: Option[Double],
  attachedMediaType: Option[String],
  attachedAssetURL: Option[String],
  created: Option[Long],
  updated: Option[Long],
  searchTags: Option[String],
  metaData: Option[String],
  returning: Option[String]
)

object AssetResponse {
  implicit lazy val assetResponseJsonFormat: Format[AssetResponse] = Json.format[AssetResponse]
}

